code/game/objects/items/weapons/implant/implanter.dm

```diff
@@ -78,3 +79,89 @@
 
 			implant = null
 			update_icon()
+
+/obj/item/implanter/installer
+	name = "cybernetic installer"
+	desc = "A medical applicator of cybernetics."
+	icon_state = "installer_empty"
+	w_class = ITEM_SIZE_NORMAL
+	matter = list(MATERIAL_PLASTIC = 5, MATERIAL_STEEL = 3)
+	var/obj/item/organ_module/mod
+	var/mod_overlay = null
+	spawn_blacklisted = TRUE
+
+/obj/item/implanter/installer/New()
+	..()
+	if(ispath(mod))
+		mod = new mod(src)
+		update_icon()
+
+/obj/item/implanter/installer/attack_self(mob/user)
+	if(!mod)
+		return ..()
+	user.put_in_hands(mod)
+	to_chat(user, SPAN_NOTICE("You remove \the [mod] from \the [src]."))
+	mod = null
+	update_icon()
+	return
+
+/obj/item/implanter/installer/update_icon()
+	if(mod)
+		icon_state = mod.mod_overlay
+	if(mod.mod_overlay == null)
+		icon_state = "installer_full"
+	if(!mod)
+		icon_state = "installer_empty"
```

Comment:
```suggestion
	if(mod)
		if(mod.mod_overlay == null)
			icon_state = "installer_full"
		else
			icon_state = mod.mod_overlay
	else
		icon_state = "installer_empty"
```

Reply:
Otherwise it will runtime when trying to check `if(mod.mod_overlay == null)` when mod is null